config/owasp/suppressions.xml

```diff
@@ -73,4 +73,8 @@
     <notes>Wait for updates on dependencies</notes>
     <cve>CVE-2021-37533</cve>
   </suppress>
+  <suppress>
+    <notes>Wait for spring boot 3 upgrades</notes>
+    <cve>CVE-2023-20883</cve>
```

Comment:
Always good to put an end date on suppressions so it forces us to review them.  E.g.   
<suppress until ="2023-07-31">

Reply:
Yes, added the suppression until. Thank you.